### ISSUE TITLE
testsuite failures in Stackage Nightly

### ISSUE THREAD

**Issue by juhp** [jgm/typst-hs#11]
Test suite failure for package typst-0.3.1.0:

```
             test/typ/math/spacing-04.typ:                FAIL (0.01s)                                                              
               Test output was different from 'test/out/math/spacing-04.out'. Output of ["diff","-u","test/out/math/spacing-04.out",
"/tmp/spacing-0444-40.actual"]:                                                                                                     
               --- test/out/math/spacing-04.out 2023-07-24 03:03:19.932092787 +0000                                                 
               +++ /tmp/spacing-0444-40.actual  2023-08-19 06:26:08.131211805 +0000                                                 
               @@ -283,96 +283,5 @@                                                                                                 
                    ]                                                                                                               
                , ParBreak                                                                                                          
                ]                                                                                                                   
               ---- evaluated ---                                                                                                   
               -{ text(body: [                                                                                                      
               -]),                                                                                                                 
               -  text(body: [                                                                                                      
               -]),                                                                                                                 
               -  math.equation(block: false,                                                                                       
               -                body: { text(body: [a]),                                                                            
               -                        text(body: [≡]),                                                                            
               -                        text(body: [b]),                                                                            
               -                        text(body: [+]),                                                                            
               -                        text(body: [c]),                                                                            
               -                        text(body: [-]),                                                                            
               -                        text(body: [d]),                                                                            
               -                        text(body: [⇒]),                                                                            
               -                        text(body: [e]),                                                                            
               -                        math.op(limits: false,                                                                      
               -                                text: "log"),                                                                       
               -                        text(body: [5]),                                                                            
               -                        math.op(text: text(body: [ln])),                                                            
               -                        text(body: [6]) },                                                                                         -                numbering: none),                                                                                   
               -  text(body: [ ]),                                                                                                  
               -  linebreak(),                                                                                                      
               -  math.equation<truncated>                                                                                          
               Use --accept or increase --size-cutoff to see full output.                                                           
               Use -p '/test\/typ\/math\/spacing-04.typ/' to rerun this test only.
```

```
             test/typ/math/spacing-02.typ:                FAIL (0.01s)
               Test output was different from 'test/out/math/spacing-02.out'. Output of ["diff","-u","test/out/math/spacing-02.out",
"/tmp/spacing-0244-98.actual"]:                                   
               --- test/out/math/spacing-02.out 2023-07-24 03:03:19.932092787 +0000
               +++ /tmp/spacing-0244-98.actual  2023-08-19 06:26:08.443205900 +0000
               @@ -110,47 +110,5 @@                         
                    ]                                                                                                               
                , ParBreak
                ]
               ---- evaluated --- 
               -{ text(body: [
               -]), 
               -  math.equation(block: false, 
               -                body: { text(body: [a]), 
               -                        text(body: [ ]), 
               -                        text(body: [b]), 
               -                        text(body: [,]), 
               -                        text(body: [a]), 
               -                        text(body: [ ]), 
               -                        text(body: [b]), 
               -                        text(body: [,]), 
               -                        text(body: [a]), 
               -                        text(body: [ ]), 
               -                        text(body: [b]), 
               -                        text(body: [,]), 
               -                        text(body: [a]), 
               -                        text(body: [ ]), 
               -                        text(body: [b]) }, 
               -                numbering: none), 
               -  text(body: [ ]), 
               -  linebreak(), 
               -  math.equation(b<truncated>
               Use --accept or increase --size-cutoff to see full output.
               Use -p '/test\/typ\/math\/spacing-02.typ/' to rerun this test only.
```

If you would the testsuite to count after resolving this, please open a PR to revert the following stackage commit.

**Comment by jgm** [comment]
Yes, I know.  I put out a fixed release an hour or so after that one...and deprecated that one on Hackage.

**Comment by jgm** [comment]
To be more precise, the problem was not with typst but with the dependent package typst-symbols, which had a bad update 0.1.3.  New versions of typst-symbols are now available, and 0.1.3 is deprecated on Hackage.  But what would be best is if stackage stayed with typst-symbols 0.1.2 as long as typst 0.3.1.0 is included.  I should have included an upper bound on typst-symbols.
